client/src/main/java/org/spine3/protobuf/KnownTypes.java

```diff
@@ -204,9 +204,6 @@ private Builder putProperties(Properties properties) {
             return this;
         }
 
-        //TODO:2016-07-07:alexander.yevsyukov: Can we do what the below method does automatically?
-        // We can compose the same map on compile time by analyzing proto files packed with Protobuf's JAR.
-
         /**
```

Comment:
Please remind me, what was the reason for not doing it the way suggested in the TODO?

Maintaining Protobuf changes on our side can be done, but it's manual work, and we have to remember about it.